src/ui/widgets.rs

```diff
@@ -6,7 +6,7 @@ use bevy::{
     ui::Val::*,
 };
 
-use super::{interaction::InteractionPalette, palette::*};
+use super::{interaction::InteractionPalette, palette::*, prelude::OnPress};
```

Comment:
```suggestion
use super::{interaction::{InteractionPalette, OnPress}, palette::*};
```

Reply:
Ran nightly cargo fmt.

Reply:
This isn't resolved by `fmt`, would need to change the import to be from `interaction` instead of `prelude` first.

Reply:
Oh, right. Thanks.